tests/run-make/fmt-write-bloat/rmake.rs

```diff
@@ -0,0 +1,16 @@
+//@ ignore-windows
```

Comment:
Suggestion: we should document why, namely:

```rs
// Reason: This test is `ignore-windows` because the `no_std` test (using `#[link(name = "c")])` 
// doesn't link on windows.
```